packages/web/pages/player/[username].tsx

```diff
@@ -427,50 +499,47 @@ export const Grid: React.FC<Props> = ({ player: initPlayer }): ReactElement => {
         onLayoutChange={(layoutItems, layouts) => {
           handleLayoutChange(layoutItems, layouts);
         }}
-        verticalCompact
         layouts={displayLayouts}
-        breakpoints={{ lg: 1180, md: 900, sm: 768, xxs: 0 }}
-        preventCollision={false}
-        cols={{ lg: 3, md: 2, sm: 1, xxs: 1 }}
+        breakpoints={{ lg: 1180, md: 900, sm: 0 }}
+        cols={{ lg: 3, md: 2, sm: 1 }}
         rowHeight={GRID_ROW_HEIGHT}
         isDraggable={!!editable}
-        isResizable={!!editable}
-        onDragStart={toggleScrollLock}
-        onDragStop={toggleScrollLock}
-        onResizeStart={toggleScrollLock}
-        onResizeStop={toggleScrollLock}
-        transformScale={1}
+        isResizable={false}
         margin={{
           lg: [30, 30],
           md: [30, 30],
           sm: [30, 30],
-          xxs: [30, 30],
         }}
         containerPadding={{
           lg: [30, 30],
           md: [20, 20],
           sm: [20, 20],
-          xxs: [15, 15],
         }}
       >
-        {currentLayoutItems.map(({ boxKey, boxType, boxMetadata }) =>
+        {currentLayoutItems.map(({ boxKey, boxType, boxMetadata }, i) =>
           boxType === BoxType.PLAYER_ADD_BOX ? (
-            <Flex key={boxKey} className="gridItem">
+            <Flex key={boxKey} className="gridItem" id={boxKey}>
```

Comment:
Why are you setting the `id`? Is it referenced by a stylesheet or something?

Also, the `Flex` tags for these two sections are the same, they can be moved to wrap the ternary and deduplicated.

Reply:
@dysbulic The `id` is needed for `getBoxKeyFromTarget` in line 166